app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,22 @@
+class ReviewsController < ApplicationController
+  def new
+  end
+
+  def create
+    @tattoo = Tattoo.find(params[:tattoo_id])
+    @review = @tattoo.reviews.new(review_params)
+    @review.user = current_user
+    if @review.save
+      redirect_to tattoo_path(@tattoo)
```

Comment:
It would be nice to have a flash message letting users know of the successful review creation!